src/DICOM.jl

```diff
@@ -158,21 +159,23 @@ function pixeldata_parse(st, sz, vr, dcm)
         dtype = UInt16
     end
     if !is(dcm,false)
+	# (0028,0010) defines number of rows
         f = lookup(dcm, (0x0028,0x0010))
         if !is(f,false)
-            xr = f.data[1]
+            xr = f.data[1][1]
```

Comment:
Thanks for merging.  I'm still learning the DICOM format, but I think you set up the `data` field to be of type `Array{Any}` to accommodate for single values read in (like Rows or Columns) or a list of numbers like Image Orientation.  I think I'm getting to the point where I might be able to do the rework in the parser, if you guys don't see problems at the output end.  I'm not sure what types of applications we are feeding and appreciate information about how people use this library.